features/FxA-89-devices-view/README.md

```diff
@@ -0,0 +1,410 @@
+
+# Devices View
+
+Feature card: https://github.com/mozilla/fxa-features/issues/8
+
+Original draft: https://docs.google.com/document/d/1VrysJ4LFPZ20IeMMmRts5l_2GUHhm890rlQhD012U_g
+
+Engineering bugs: https://waffle.io/mozilla/fxa?milestone=FxA-89:%20devices%20view
+
+## Stories
+
+* So that I can best ensure the integrity of my Sync setup,
+  as a Firefox Sync user,
+  I want to see information about the devices
+  connected to my Firefox Account.
+* So that I can obtain a device overview more quickly,
+  I want the devices connected to my account
+  to have recognizable names by default.
+* I want to easily see the type
+  and browser details of the each device, 
+* So that I can better tell what device I am currently using,
+  I want it to be easily distinguishable from the other items.
+* So that I can easily detect when a device is no longer active,
+  I want to be able to see when my devices last authenticated.
+
+
+## Hypothesis
+
+1. If we add a device view in settings
+   for Firefox Account users with multiple devices,
+   then **we will decrease the number of malicious connections**
+   because we will observe users managing and disconnecting
+   suspicious devices which they cannot presently do.
+2. If we add a device view in settings
+   for Firefox Account users with multiple devices,
+   then **we will decrease the number of old/lost devices
+   still connected to an account**
+   because we will observe users managing their devices.
+3. Less importantly,
+   if we present users with download links for Firefox
+   when they don’t have a 2nd device,
+   **we will increase multi-device users**
+   because we will generate downloads from the device management view.
+
+## Metrics
+
+### Key Success Metric
+
+We will know our hypotheses to be true when we measure that users are disconnecting old, lost and unknown devices.
+
+We will measure this through the confirmation dialogue after users click to disconnect a device.
+
+### Secondary Metrics
+
+* Number of device disconnections per day.
+  * Categorized by traffic source (e.g email, settings via hamburger menu)
+* Percentage of clicks on "missing or duplicate devices?" link.
+  * # of users that clicked it / # of users that saw device view
+* Percentage refresh rate
+  * # of users that clicked refresh / # of users that saw device view
+* Clicks on the "Connect..." button
+  * # of users that clicked it / # of users that saw the button
+* Number of mobile installs generated from the page.
+  * From mobile team metrics via adjust.com
+* Number of clicks on "I did not disconnect this device" in email
+  * From mobile team metrics via adjust.com
+
+
+## UX
+
+### Assets
+
+Use these icons for desktop, phone, and tablet device types:
+
+<img src="screenshot-of-device-icon-assets.png" />
+
+TODO: @ryanfeeley to provide raw asset files to add here.
```

Comment:
ping @ryanfeeley; please provide the asset files directly, I assume we want then as the raw svg.